lang/README.md

```diff
@@ -0,0 +1,29 @@
+if you want add new language;
+
+  Create `<language_you_added.sh>` file, at that dir.
+
+It name will be such as english.sh
+
+and run **tetris.sh** as:
+```text
+./tetris.sh -l english
+```
+Tetris script will automatically identify the local variables you added.
+
+
+EXAMPLE `language_you_added.sh` file:
+```bash
+i18n_lines_completed="Lines completed: ";
```

Comment:
Sorry, just noticed: terminating semicolon is not needed here.

Reply:
different PR is suitable